code/modules/mob/living/carbon/human/human.dm

```diff
@@ -835,36 +841,70 @@
 		return TRUE
 	return FALSE
 
-/mob/living/carbon/human/buckle_mob(mob/living/M, force = FALSE, check_loc = TRUE)
+/mob/living/carbon/human/proc/can_be_firemanned(mob/living/carbon/C)

```

Comment:
clarify via argname that arg is the target and src is user

```suggestion
/mob/living/carbon/human/proc/can_be_firemanned(mob/living/carbon/target)
```

Reply:
```DM
/mob/living/carbon/human/proc/can_be_firemanned()
    return (ishuman(src) && !(mobility_flags & MOBILITY_STAND))
```

Although this could probably just be a define since it's on the human mob.